test/mocha.opts

```diff
@@ -1 +1,2 @@
 --timeout 1200s
+--exit
```

Comment:
Why are we adding the `exit` parameter?

Reply:
See the docs: https://mochajs.org/#--exit----no-exit

This is neccessary, because the exit-behaviour of mocha changed from v3 to v4.

quote: "The easiest way to “fix” the issue is to simply pass --exit to the Mocha process."

This is a quick solution for upgrading to mocha version 4 without having the problem, that the tests don't exit properly. Maybe we should have a closer look on these tests, to find out why they are not exiting, but not as part of this PR

Reply:
KK, thanks!